registry.json

```diff
@@ -2880,5 +2880,35 @@
       "source": "https://github.com/hashgraph",
       "documentation": "https://docs.hedera.com"
     }
+  },
+  {
+    "id": "nativeinjective",
+    "name": "NativeInjective",
+    "displayName": "Native Injective",
+    "coinId": 10000060,
+    "symbol": "INJ",
+    "decimals": 18,
+    "blockchain": "Cosmos",
+    "derivation": [
+      {
+        "path": "m/44'/60'/0'/0/0"
```

Comment:
Just to make sure: Eth derivation path is reused here, even though this is a Cosmos chain, correct?

Reply:
Yes, Eth derivation path is used for Native Injective.  https://docs.injective.network/learn/basic-concepts/accounts/#injective-accounts